examples/multiple_windows/lib/app/main_window.dart

```diff
@@ -0,0 +1,255 @@
+    windowManagerModel.add(
+      KeyedWindow(
+        isMainWindow: true,
+        key: UniqueKey(),
+        controller: mainController,
+      ),
+    );
+  }
+
+  final WindowManagerModel windowManagerModel = WindowManagerModel();
+  final WindowSettings settings = WindowSettings();
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewAnchor(
+      view: ListenableBuilder(
+        listenable: windowManagerModel,
+        builder: (BuildContext context, Widget? _) {
+          final List<Widget> childViews = <Widget>[];
+          for (final KeyedWindow window in windowManagerModel.windows) {
+            if (window.parent == null && !window.isMainWindow) {
+              childViews.add(
+                WindowControllerRender(
+                  controller: window.controller,
+                  key: window.key,
+                  windowSettings: settings,
+                  windowManagerModel: windowManagerModel,
+                  onDestroyed: () => windowManagerModel.remove(window.key),
+                  onError: () => windowManagerModel.remove(window.key),
+                ),
+              );
+            }
+          }
+
+          return ViewCollection(views: childViews);
+        },
+      ),
+      child: Scaffold(
+        appBar: AppBar(title: const Text('Multi Window Reference App')),
+        body: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Expanded(
+              flex: 60,
+              child: SingleChildScrollView(
+                scrollDirection: Axis.vertical,
+                child: _WindowsTable(windowManagerModel: windowManagerModel),
+              ),
+            ),
+            Expanded(
+              flex: 40,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: [
+                  ListenableBuilder(
+                    listenable: windowManagerModel,
+                    builder: (BuildContext context, Widget? child) {
+                      return _WindowCreatorCard(
+                        selectedWindow: windowManagerModel.selected,
+                        windowManagerModel: windowManagerModel,
+                        windowSettings: settings,
+                      );
+                    },
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _WindowsTable extends StatelessWidget {
+  const _WindowsTable({required this.windowManagerModel});
+
+  final WindowManagerModel windowManagerModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListenableBuilder(
+      listenable: windowManagerModel,
+      builder: (BuildContext context, Widget? widget) {
+        return DataTable(
+          showBottomBorder: true,
+          onSelectAll: (selected) {
+            windowManagerModel.select(null);
+          },
```

Comment:
I don't have this app running in front of me anymore so this is from memory, but I think this selection feature was broken. I wasn't quite sure what it was for, and it didn't seem possible to select multiple windows at the same time.